Core/ELF/ParamSFO.cpp

```diff
@@ -276,3 +280,34 @@ void ParamSFOData::ValueData::SetData(const u8* data, int size)
 	u_size = size;
 }
 
+std::string ParamSFOData::generateFakeID(std::string filename)
```

Comment:
Convention is for Capital first letters, and new functions to have curly on the same line.

-[Unknown]